libraries/src/Mail/MailTemplate.php

```diff
@@ -283,7 +285,7 @@ public function send()
 			$this->mailer->addReplyTo($this->replyto->mail, $this->replyto->name);
 		}
 
-		$path = JPATH_ROOT . '/' . $config->get('attachment_folder') . '/';
+		$path = Path::check(JPATH_ROOT . '/' . $config->get('attachment_folder') . '/');
```

Comment:
```suggestion
		$path = Path::check(JPATH_ROOT . '/' . str_replace('../', '', $config->get('attachment_folder') . '/'));
```

@PhilETaylor is this ok?

Reply:
In my exact case - probably - but remember Im not a hacker, Im sure a hacker has better skills than I do and can use more than `../` - it also a tacky way of doing it. The value should be validated into and validated out of the db before even being used in a file path.

Reply:
No this is not good enough, as its not the `attachment_folder` Im hacking anymore, Im hacking the filename in the dropdown and making it relative :) so no.

Reply:
Yes and this is what I mean, it is a different exploit than the one I am fixing here because here I am fixing the attachment_folder.

Reply:
Do i get 2 brownie points for this then then...If only Joomla had some kind of dedicated security team....